### PR TITLE
Make trees scale by height!

### PR DESCRIPTION
**Pull request by mortizzle**

### CODE REVIEW

**Comment by ScottHolden** [comment]
image
(Stealing Darran's image :P)

File: src/Trains.NET.Rendering/Terrain/TerrainColourLookup.cs [hunk 1]
```diff
@@ -6,8 +6,9 @@ namespace Trains.NET.Rendering
     public static class TerrainColourLookup
     {  
         public static Color DefaultColour => Colors.Green;
+        private static Color WaterToLandTransition = Colors.LightYellow;
 
-        private static readonly List<Color> s_heightOrderedColours = new List<Color> {
+        private static readonly List<Color> _heightOrderedColours = new List<Color> {
```
Comment (davidwengier) [inline src/Trains.NET.Rendering/Terrain/TerrainColourLookup.cs:11]: Why did you rename `s_heightOrderedColours` to `_heightOrderedColours`? now I'm just gonna get squiggles and have to rename it back. Did you not get squiggles after renaming it locally?

File: src/Trains.NET.Rendering/LayerRenderer/Bases/LayoutRenderer.cs [hunk 2]
```diff
@@ -40,40 +44,85 @@ public void Render(ICanvas canvas, int width, int height, IPixelMapper pixelMapp
...
+                    }
 
                     _renderer.Render(canvas, entity);
                 }
 
                 canvas.Restore();
             }
         }
+
+        private float GetScaleForTerrainHeight(int column, int row)
```
Comment (davidwengier) [inline src/Trains.NET.Rendering/LayerRenderer/Bases/LayoutRenderer.cs:96]: I'm surprised you think `GetScaleForTerrainHeight` belongs on the base layer renderer.
